tests/unit/test_plot_spectrum.py

```diff
@@ -0,0 +1,78 @@
+import os
+import pytest
+import numpy as np
+import pandas as pd
+from . import context
+from plot_spectrum import main as plot_spectrum
+
+
+import tempfile
+
```

Comment:
maybe add test to check if return values are ok?

Reply:
added. Noticed also possible problem with non-unique filenames. Solved in new commit